uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-custom-work-pattern-modal.component.js

```diff
@@ -107,12 +107,12 @@ define([
     }
 
     /**
-     * Loads the Job Contract Revision Change Reasons and indexes by `value`
+     * Loads the Leave and Absences Work Pattern Change Reason and indexes by `value`
      *
      * @return {Promise}
      */
     function loadJobContractRevisionChangeReasons () {
-      return OptionGroup.valuesOf('hrjc_revision_change_reason')
+      return OptionGroup.valuesOf('hrleaveandabsences_work_pattern_change_reason')
         .then(function (reasons) {
           vm.changeReasons = reasons;
         });
```

Comment:
Since you made changes to the JS files, you need to update the distributive/minified files. You can do that with `gulp build --ext uk.co.compucorp.civicrm.hrleaveandabsences`.

Also, since you now made the FE changes part of this PR, please update its description with a gif showing the changes working

Reply:
Alright, i will.